test/uk/gov/hmrc/digitalservicestaxfrontend/data/ValidatedTypeTests.scala

```diff
@@ -66,9 +68,13 @@ class ValidatedTypeTests extends FlatSpec with Matchers with ScalaCheckDrivenPro
     }
   }
 
-  it should "correctly add 9 months and one day to a period" in {
-    forAll { period: Period =>
-      period.paymentDue shouldEqual period.end.plusMonths(9).plusDays(1)
+  it should "return an appropriate paymentDue date" in {
+    forAll { (p1:Period, p2:Period) =>
+      List(p1,p2).sorted shouldEqual List(p1, p2).sortBy(_.paymentDue)
+    }
+    forAll { p:Period =>
+      p.paymentDue.isAfter(p.end.plusMonths(9)) &&
+        p.paymentDue.isBefore(p.end.plusMonths(10).withDayOfMonth(2)) shouldBe true
```

Comment:
as you have defined an `Order` you could just import `cats.syntax.order._` and then use `p.paymentDue < p.end.plusMonths(9)`

Reply:
Sorry, my mistake, that would be if you had the Order instance for LocalDate. You could do this too if you want, something like this - 

```
implicit val dateOrder = cats.Order.from((a: LocalDate, b: LocalDate) => a.toEpochDay compare b.toEpochDay
```

probably not worth bothering with though for 1 case, unless you wanted to use that elsewhere....